qmlti/lti/LTI_Data_Connector_oci.php

```diff
@@ -671,6 +671,97 @@ public function Resource_Link_Share_Key_delete($share_key) {
 
```

Comment:
Thank you for raising this concern! After talking with Steve about this, we decided to move development solely to PDO (which is the only db platform currently in use at the moment). The rest are a holdover from the LTI tool created in the past, and are not planned to be supported in the future.